src/v0/destinations/klaviyo/transform.js

```diff
@@ -215,45 +218,23 @@ const groupRequestHandler = (message, category, destination) => {
     delete profile.$id;
     profile._id = getFieldValueFromMessage(message, 'userId');
   }
-  const responseArray = [];
 
-  const payload = {
+  // const subscribeProfile = JSON.parse(JSON.stringify(profile));
```

Comment:
```suggestion

```

Reply:
Removed this and membership from LIST_CONF.